src/services/apiClient.ts

```
@@ -0,0 +1,33 @@
+import { FETCH_ERROR_MESSAGE, DEFAULT_ERROR_MESSAGE } from '../constants/errorMessages';
+
+interface ApiClientProps {
+  method: 'GET' | 'POST' | 'PATCH' | 'DELETE';
+  URI: string;
+  body?: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)